src/azure-cli/azure/cli/command_modules/security/_help.py

```diff
@@ -59,6 +59,37 @@
         az security alert update -g "myRg" --location "centralus" -n "alertName" --status "activate"
 """
 
+helps['security atp'] = """
+type: group
+short-summary: View and manage Advanced Threat Protection settings.
+"""
+
+helps['security atp storage'] = """
+type: group
+short-summary: View and manage Advanced Threat Protection settings for storage accounts.
+"""
+
+helps['security atp storage show'] = """
+type: command
+short-summary: Display Advanced Threat Protection settings for a storage account.
+examples:
+  - name: Retrieve Advanced Threat Protection settings for a storage account on a subscription scope.
+    text: >
+        az security atp storage show --resource-group MyResourceGroup --storage-account-name MyStorageAccount
+"""
+
+helps['security atp storage update'] = """
+type: command
+short-summary: Toggle status of Advanced Threat Protection for a storage account.
+examples:
+  - name: Enable Advanced Threat Protection for a storage account on a subscription scope.
+    text: >
```

Comment:
One more thing, it would be better to follow standard YAML syntax use  Folded style `|` instead of `>`.
- Folded style removes single newlines within the string (but adds one at the end, and converts double newlines to singles.
- Literal style turns every newline within the string into a literal newline, and adds one at the end